Styles/mobilemain.css

```diff
@@ -24,6 +24,9 @@
         text-align: center;
         border-top: none;
     }
+    nav ul li a {
+        border: none;
+    }

```

Comment:
You also need to remove the transparent border so that the links to shift up/down when hovering